### ISSUE TITLE
Profession specific drops and visibility in large groups.

### ISSUE THREAD

**Issue by Cabro** [LightsHope/server#178]
There are some items that drop and only certain crafters should be able to see them.
For example the Major Mana Potion recipe from Gandling in Scholomance for alchemists, the Tailoring Pattern from Pyron for tailors etc.
The bug follows: if you are in the group and the recipe/pattern drops, it does not show up in loot popups, you will only see it when you check yourself the corpse.
Here's the catch: it should only popup for the ones that can actually use it.
So if you have 1 alchemist in group and the major mana potion recipe drops, it should only show in popup for the alchemist, even if he's not the one that loots first the mob to trigger the popup.
The same rule applies quest items. For example if someone that has the scourgestone trinket equipped, and the others don't, and they drop. Only the one(s) with trinket equipped should have popup for the scourgestones (if applicable to show up in popup).

**Comment by GuybrushGit** [comment]
Not all patterns are affected by this: http://web.archive.org/web/20060428123422/http://www.thottbot.com:80/?i=21483
Need sources for the ones that are.

**Comment by Memnok1** [comment]
https://web.archive.org/web/20060516214926/http://wow.allakhazam.com:80/db/item.html?witem=13501
Comment by: SchismFierna confirms that major mana potion can drop even if you don't have alchemy as your profession.

Can't find any info that Cloak of Fire only drops for tailors.

**Comment by Cabro** [comment]
The Major Mana Potion recipe drop from Gandling is 100% affected by this rule. Not only you shouldn't get the loot popup, but you shouldn't even see it in loot window if you're not alchemist. I have stumbled upon this pattern many times on retail and it was like this many times.
As a general rule, the recipes that are BoP should be bound to this rule. I might have been wrong about the Cloak of Fire pattern, since it's not BoP.

**Comment by Memnok1** [comment]
This is a very confusing one. I can't find any information about any recipe or pattern only dropping for one class, yet it makes a sort of sense that a BoP recipe would only drop for that profession since no one else has a use of it and would only serve for people to steal it from an alchemist.

The problem with this though is that in patch 1.3 http://wowwiki.wikia.com/wiki/Patch_1.3.0 the Robe of the Archmage and Robe of the void were changed to no longer be soulbound, but there's no mention that players without tailoring will be able to roll/see the item drop. This is something that should've been in the patch notes imo.

The other thing is that in the Comment by SchismFierna he does not mention anything akin to "Warning, if you don't have alchemy you can't roll on or see this recipe" which would have been very interesting for anyone to know if they were planning on changing/dropping their profession, every time i see a comment about it dropping it usually mentions "It dropped for us" meaning the whole group, not just one person.

Then again, anyone can see and purchase this recipe from the vendor outside Scholomance, why restrict the drop from Gandling to only be visible to Alchemists when anyone can do the quest and get access to the recipe. 

If you have any other info or sources please bring them forward

**Comment by Cabro** [comment]
https://us.battle.net/forums/en/wow/topic/8197761237

**Comment by Cabro** [comment]
Did you guys read the last link i gave?

**Comment by Memnok1** [comment]
Yes, but that info is from MoP, i remember there being recipies that are only visible to the player with a profession in TBC, specifically sunwell. But i have no memory of it being so in vanilla and i can not find any info about it, be it patch notes or otherwise.

**Comment by Cabro** [comment]
Likewise here, i remember this from TBC also, i didnt play beyond that patch much and it was behaving like that then. There are also NO mentions of this being changed in MoP either, so by deduction it should've been like this in vanilla also.
